engine/build.gradle

```diff
@@ -1,5 +1,6 @@
 plugins {
     id 'com.android.library'
+	id 'kotlin-android'
```

Comment:
Please, make sure the indenting is consistent. This seems like a space vs tab issue.